pvl/lexer.py

```diff
@@ -89,9 +41,9 @@ def lex_preserve(char: str, lexeme: str, preserve: dict) -> tuple((str, dict)):
     """
     # print(f'in preserve: char "{char}", lexeme "{lexeme}, p {preserve}"')
```

Comment:
print statements like these if useful should be converted into logging calls rather than commented out code

Reply:
Fair enough.  These sorts of things could be worked in a cleanup pass through the code, but its not really relevant to this PR effort.